src/vorta/views/extract_dialog.py

```diff
@@ -20,21 +22,26 @@ def __init__(self, fs_data, archive):
         nested_file_list = nested_dict()
         self.selected = set()
 
-        def parse_line(line):
-            size, modified, full_path = line.split("\t")
-            size = int(size)
-            dir, name = os.path.split(full_path)
-
+        def parse_json_line(line):
+            data = json.loads(line)
+            size = data["size"]
+            # python >= 3.7
+            # modified = datetime.fromisoformat(data["mtime"]).ctime()
+            # python < 3.7
+            try:
+                modified = datetime.strptime(data["mtime"], "%Y-%m-%dT%H:%M:%S.%f").ctime()
```

Comment:
Actually it's still fast even when parsing the date. Tried with 40k files.

Reply:
I don't recall now, but I think I was trying to get it into a datetime object so it would be easier to manipulate how it was displayed on the gui, if ever needed.

Reply:
The time string from json is a bit ugly with MS included. So let's just use `ctime()` for now. Maybe in the future there could be a setting for a default time format or something.